atomic_reactor/build.py

```diff
@@ -183,6 +185,29 @@ def set_df_path(self, path):
         for image in dfp.parent_images:
             self.parent_images[image] = None
 
+        # validate user has not specified COPY --from=image
+        builders = []
+        for stmt in dfp.structure:
+            if stmt['instruction'] == 'FROM':
+                match = re.match(r'\S+\s+as\s+(\S+)', stmt['value'], re.I)
+                builders.append(match.group(1) if match else str(len(builders)))
```

Comment:
This is so clever, it could be admittance to a secret club.

Reply:
Oh, that's not a compliment is it?

Reply:
It's not an insult either. I thought it was cool :)

Reply:
Actually a bit of the cleverness wasn't needed at all. I hope it's clearer now.